packages/react-components/lib/doors/door-panel.tsx

```diff
@@ -39,19 +39,26 @@ const useStyles = makeStyles((theme) => ({
   },
   grid: {
     padding: '1rem',
+    maxHeight: '40vh',
+    overflowY: 'scroll',
   },
   doorLabelOpen: {
     backgroundColor: theme.palette.success.main,
+    padding: '0 0.2rem',
   },
   doorLabelClosed: {
     backgroundColor: theme.palette.error.main,
+    padding: '0 0.2rem',
   },
   doorLabelMoving: {
     backgroundColor: theme.palette.warning.main,
+    padding: '0 0.2rem',
   },
   cellPaper: {
     padding: '0.5rem',
-    backgroundColor: theme.palette.info.light,
+    backgroundColor: theme.palette.common.white,
```

Comment:
use theme colors to avoid bugs

![image](https://user-images.githubusercontent.com/45189696/137828800-35f36108-b289-496e-88e4-17f67b4d8602.png)